translate/storage/test_php.py

```diff
@@ -1182,22 +1182,62 @@ def phpparse(self, phpsource):
         return self.StoreClass(dummyfile)
 
     def test_plurals(self):
-        phpsource = r"""<?php
-return [
-    'welcome' => 'Welcome to our application',
-    'apples' => 'There is one apple|There are many apples',
-];
-"""
+        phpsource = open("./laravel/plurals.php").read()
```

Comment:
You can not make assumptions about current working directory. Either derive the filename from ``__file__`` and make sure the test files are included in the package, or inline it as it was done before.